src/pages/deriv-go/_start-deriv-go.tsx

```diff
@@ -37,64 +38,45 @@ const StyledHeader = styled(Header)`
         padding: 0 20px;
     }
 `
-const StyledLocalizedLink = styled(LocalizedLink)`
-    text-decoration: none;
-    color: rgba(255, 68, 79, 1);
+const StartDerivGo = () => {
+    const [is_mobile, setMobile] = useState(false)
+    const data = useStaticQuery(query)
 
-    &:hover {
-        text-decoration: underline;
+    const handleResizeWindow = () => {
+        setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)
     }
-`
 
-const StartDerivGo = () => {
-    const data = useStaticQuery(query)
+    useEffect(() => {
+        setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)
+        window.addEventListener('resize', handleResizeWindow)
+    })
+    const stepsData: React.ComponentProps<typeof StepperView>['items'] = useMemo(
+        () => [
+            {
+                title: 'Sign in to your Deriv account. If you don’t have one, sign up for free; then create a Deriv real account.',
+                image: data['sign_up'],
+                alt: <Localize translate_text="Sign in" />,
+            },
+            {
+                title: 'Fund your Deriv real account with your preferred payment method.',
+                image: data['fund'],
+                alt: <Localize translate_text="Fund your Deriv" />,
+            },
+            {
+                title: 'Download the app and trade anytime, anywhere.',
+                image: data['trading'],
+                alt: <Localize translate_text="Download the app" />,
+            },
+        ],
+        [data, is_mobile],
```

Comment:
Do we need `is_mobile` as a dependency here?

Reply:
I have removed and updated to other useEffect